device/device.go

```diff
@@ -348,13 +348,13 @@ func (d *device) SelectInterfaces() int {
 func (d *device) SetInterfaces(i Interfaces) error {
 	var (
 		answer string
-		num    int = 0
-		s      int = 0
-		device     = []string{"eth0", "wlan0"}
+		num    = 0
```

Comment:
__[gometalinter]__
* [ineffassign] ineffectual assignment to num

<!-- Comment by [SideCI](https://sideci.com) -->